lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -114,41 +104,61 @@ def search(
 
         logger.debug(json.dumps(response, indent=2))
 
-        page_results = self._parse_search_results(response)
+        # Should these 2 variables be bound or unbound?
```

Comment:
Do you mean should they be instance variables? 

In general I prefer these kind of helper methods to return values rather than modifying state, otherwise you get variables that may or may not be defined depending on when you access them, which adds extra complexity